### PR TITLE
Added new parameters requested by Matt Derry

### PR DESCRIPTION
**Pull request by Coldpawz**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -195,6 +207,18 @@
                             {
                                 "name": "Values:ApiXFunctionsKey",
                                 "value": "[parameters('apiXFunctionsKey')]"
+                            },
+                            {
+                                "name": "ConfigNames",
+                                "value": "[parameters('configNames')]"
+                            },
+                            {
+                              "name": "environmentName",
```
Comment (adam230594) [inline azure/template.json]:
```suggestion
                              "name": "EnvironmentName", 
```

File: azure/template.json [hunk 2]
```diff
@@ -195,6 +207,18 @@
...
                                 "name": "Values:ApiXFunctionsKey",
                                 "value": "[parameters('apiXFunctionsKey')]"
+                            },
+                            {
+                                "name": "ConfigNames",
+                                "value": "[parameters('configNames')]"
+                            },
+                            {
+                              "name": "environmentName", 
+                              "value": "[parameters('EnvironmentName')]"
```
Comment (adam230594) [inline azure/template.json]:
```suggestion
                              "value": "[parameters('environmentName')]"
```

File: azure/template.json [hunk 3]
```diff
@@ -38,6 +38,18 @@
         "appServicePlanInstances": {
             "type": "int",
             "defaultValue": 2
+        },
+        "configNames": {
+            "type": "string",
+            "defaultValue": "SFA.DAS.Campaign.Functions"
+        },
+        "environmentName": {
```
Comment (adam230594) [inline azure/template.json]: Can we move this to the top

File: azure/template.json [hunk 4]
```diff
@@ -8,6 +8,14 @@
                 "description": "Short name of the environment. Used for the name of resources created"
             }
         },
+        "environmentName": {
+            "type": "string",
+            "defaultValue": ""
```
Comment (adam230594) [inline azure/template.json]: Remove default value

File: azure/template.json [hunk 5]
```diff
@@ -38,6 +46,10 @@
         "appServicePlanInstances": {
             "type": "int",
             "defaultValue": 2
+        },
+        "configurationStorageConnectionString": {
+            "type": "string",
+            "defaultValue": ""
```
Comment (adam230594) [inline azure/template.json]: remove default value

File: azure/template.json [hunk 6]
```diff
@@ -38,6 +45,9 @@
         "appServicePlanInstances": {
             "type": "int",
             "defaultValue": 2
+        },
+        "configurationStorageConnectionString": {
+            "type": "string",
```
Comment (EwanNoble) [inline azure/template.json]: securestring please

**Pull request merged**